cluster-provision/gocli/cmd/run.go

```diff
@@ -598,35 +614,30 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 			if err != nil {
 				return fmt.Errorf("parsing proxy settings for node %s failed", nodeName)
 			}
-			success, err = docker.Exec(cli, nodeContainer(prefix, nodeName), []string{"/bin/bash", "-c", fmt.Sprintf("cat <<EOF >/scripts/docker-proxy.sh %s", proxyConfig)}, os.Stdout)
-			if err != nil {
-				return fmt.Errorf("write failed for proxy provision script for node %s", nodeName)
+
+			if _, err = sshClient.JumpSSH(fmt.Sprintf("cat <<EOF > ~/scripts/docker-proxy.sh %s", proxyConfig), false, true); err != nil {
```

Comment:
Should this be executed in context of the VM?

Reply:
Yes, as the script on the vm is not modified properly, its a template. this script also doesn't exist on the gocli it exists only on the node container
so the other option is to run this on the node container then scp it again. it my opinion its better to just run this on the vm since the script is already scp'd in the initial scp